### DISCUSSION TITLE
I need all members across all channels

### DISCUSSION THREAD

**Discussion by jasnicaa** [GetStream/stream-chat-flutter#823] [Q&A] [answered]
Hi guys, I can see your implementation in the channel list view to get this, but is there an easier way? I have this issue where I want to start a new chat and the user can select a member, but I have x amount of channels and it is listing only from one of them if I listen to a stream for members. Any suggestions?

**Comment by imtoori** [comment, marked as answer]
mmm I think you want to see all users in the chat, right?
We have UsersBloc and UserListView widgets for that, make sure to check it out

**Reply by jasnicaa** [reply]
I want to see all users through all channels. As I can see in the docs UserListView is showing a list of users in a Channel, but I need to get all channels and get each user in the channel and show them. I guess I will need to get first all channels and then get them from each..

**Reply by imtoori** [reply]
that doesn't sound a good solution
You could get all users in the whole application 

https://getstream.io/chat/docs/sdk/flutter/stream_chat_flutter/user_list_view/

here you can find a simple example for showing all users

**Reply by jasnicaa** [reply]
yeah mine solution is pretty expensive that way, but if the user list view returns what I want that's a solution. thanks
